From e8255960e61ff2bfef6b7e7e2f4038985a8d52bb Mon Sep 17 00:00:00 2001
From: Shubhrajyoti Datta <shubhrajyoti.datta@xilinx.com>
Date: Mon, 2 Mar 2020 12:28:28 +0530
Subject: [PATCH 1250/1851] clk: zynqmp: Fix missing max_div description in
 kernel-doc format

commit 7b2bf9e9e9a4ed043bed8331aa9c353d65796b82 from
https://github.com/Xilinx/linux-xlnx.git

Fix missing max_div description in kernel-doc format reported by kernel-doc
warning:
warning: Function parameter or member 'max_div' not described in
zynqmp_clk_divider

Signed-off-by: Shubhrajyoti Datta <shubhrajyoti.datta@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/clk/zynqmp/divider.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/zynqmp/divider.c b/drivers/clk/zynqmp/divider.c
index 26c25b66b138..7613678cdc89 100644
--- a/drivers/clk/zynqmp/divider.c
+++ b/drivers/clk/zynqmp/divider.c
@@ -35,6 +35,7 @@
  * @is_frac:	The divider is a fractional divider
  * @clk_id:	Id of clock
  * @div_type:	divisor type (TYPE_DIV1 or TYPE_DIV2)
+ * @max_div:	Maximum divisor value allowed
  */
 struct zynqmp_clk_divider {
 	struct clk_hw hw;
-- 
2.31.1

